app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/model/CalendarItem.kt

```diff
@@ -70,23 +58,14 @@ data class CalendarItem(
         // remove lecture codes in round or square brackets e.g. (IN0003), [MA0902]
         return Pattern.compile("[(\\[][A-Z0-9.]+[)\\]]")
             // remove type of lecture (VO, UE, SE, PR) at the end of the line
-            .matcher(
-                Pattern.compile(" (UE|VO|SE|PR)$")
-                    .matcher(title)
-                    .replaceAll("")
-            )
-            .replaceAll("")
-            .trim { it <= ' ' }
+            .matcher(Pattern.compile(" (UE|VO|SE|PR)$").matcher(title).replaceAll("")).replaceAll("").trim { it <= ' ' }
```

Comment:
I think extracting this into two variables (each of them only doing one of the replacements) might be more readable instead of this way